libopenage/CMakeLists.txt

```diff
@@ -139,6 +145,11 @@ get_config_option_string()
 configure_file(config.h.in ${CMAKE_CURRENT_SOURCE_DIR}/config.h)
 configure_file(config.cpp.in ${CMAKE_CURRENT_SOURCE_DIR}/config.cpp)
 
+# default homebrew/cellar Qt5 installation
+if(APPLE)
+	find_path(QTPLATFORM_INCLUDE_DIRS QtPlatformHeaders "/usr/local/Cellar/qt5/*/include")
```

Comment:
Sounds like I could work. I confirm that even though I installed to `/usr/local/Cellar` using `brew install qt5`: I see that brew has created also a symlink in `/usr/local/opt/qt5`:

```
lrwxr-xr-x 1 me admin 19 Oct 15 17:05 qt5 -> ../Cellar/qt5/5.6.2
```